src/client/components/Buttons/Button/Button.scss

```diff
@@ -8,14 +8,15 @@
   display: flex;
   font-family: $font-family-1;
   font-weight: 600;
-  line-height: 0;
```

Comment:
Now that this line is deleted, the default value for line-height is `normal`.
For some reason, on my laptop screen it doesn't affect the alignment:

https://github.com/openforis/fra-platform/assets/41337901/4e2bd791-bfc3-4a3e-9560-9f21ce74ef9a

But on a big screen it behaves differently and this happens 🤔 : 

https://github.com/openforis/fra-platform/assets/41337901/ffe0f1f2-de1c-4237-bd9a-2c37aa5e17c4

It's slightly off and changing the line-height to a value here would affect the other buttons:

https://github.com/openforis/fra-platform/assets/41337901/e6ef50ed-dd48-40dd-bceb-99e104824f4b